backend/src/main/java/gov/cdc/usds/simplereport/api/model/accountrequest/AccountRequest.java

```diff
@@ -33,6 +33,11 @@
   @JsonProperty private String facilityTypeOther;
   @JsonProperty private String organizationName;
   @JsonProperty @NotNull private String facilityName;
+
+  @JsonProperty @NotNull
```

Comment:
can you just confirm that we need facility phone # to be non-null?

Reply:
@timothybest-usds to confirm, but I'm fairly certain phone number is required so should be non-null.

Reply:
It's required in the facility form: 
<img width="400" alt="Screen_Shot_2021-04-19_at_12_48_02_PM" src="https://user-images.githubusercontent.com/5249443/115294550-961c0e80-a10d-11eb-984e-9ec2d808e944.png">

Reply:
Yes facility phone number is required and should be non-null